checks/checks.go

```diff
@@ -0,0 +1,9 @@
+package checks
+
+import "github.com/TimothyStiles/poly/transform/reverse"
+
+// IsPalindromic accepts a sequence of even length and returns if it is
+// palindromic. More here - https://en.wikipedia.org/wiki/Palindromic_sequence
+func IsPalindromic(sequence string) bool {
+	return sequence == reverse.Complement(sequence)
+}
```

Comment:
I think we might want to merge this into a utils package with basic complementation and reverse stuff. Thoughts?

Reply:
Also I don't know how to fix this, but if I remember correctly you code copy paste rather than the copy-paste-file then remove-content, you remove the original contributors "ownership" or whatever git tracks. I don't care, but other folks might.

Reply:
> I think we might want to merge this into a utils package with basic complementation and reverse stuff. Thoughts?

See -> https://github.com/TimothyStiles/poly/pull/174#discussion_r661935919

Reply:
> Also I don't know how to fix this, but if I remember correctly you code copy paste rather than the copy-paste-file then remove-content, you remove the original contributors "ownership" or whatever git tracks. I don't care, but other folks might.

I worried about this but GitHub seemed to be tracking this pretty well and it may be a new feature of git. I'll look into this before merge.